src/components/icons/controller.icons.ts

```diff
@@ -101,9 +101,45 @@ const getFile = async (req: Express.Request, res: Express.Response, next: Expres
   }
 }
 
+const getFont = async (req: Express.Request, res: Express.Response, next: Express.NextFunction) => {
+  try {
+    const theme = req.query.theme as iconsTheme
+    const svgField = getAppropriateSVGField(theme) as svgFieldsInDB
+    const data:getIcon = req.body
+    const { icons, customizations } = data
+    const setOfIcons = await iconsServices.getSetOfIcons(icons, `-_id name ${svgField}`)
+    const svgStrings = {}
+    for (const icon of setOfIcons) {
+      const iconCustomizer = new SvgFactory(icon[svgField]!, customizations, !!customizations)
+      const customizedSVG = iconCustomizer.finalizeIcon()
+      svgStrings[icon.name] = customizedSVG
+    }
+    const srcFolderTimestamp = Math.floor(Date.now())
+    const imageFactoryData:CustomizedIconsPayload = { ...data, exportAs: 'svg' }
+    const imageCreator = new ImageFactory(imageFactoryData, svgStrings, srcFolderTimestamp, false)
+    await imageCreator.generateTheIconsPack()
+    let outlined = false
+    if (theme === 'outlined') {
+      outlined = true
```

Comment:
we can use enums instead of these conditions

Reply:
I will merge it, and fix it with other APIs in a separate PR.